src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
+    LOG_NONE        = 0,
+    LOG_NET         = (1 <<  0),
```

Comment:
It's just that the LOG_ prefix is very common, so it's easy to get namespace collisions. Scoped enums avoid that, the only way any of their members can collide is when the name of the type conflicts, which would be an error in the first place.
Then again if this introduces extra C++ casting uglyness in all usages let's leave it like this...